### ISSUE TITLE
ComputerCraft - Mining/Felling turtle recipe not working

### ISSUE THREAD

**Issue by NS4W** [SquidDev-CC/CC-Tweaked#614]
## Useful information to include:
 - Minecraft version
 - CC: Tweaked version
 - Logs: These will be located in the `logs/` directory of your Minecraft instance. Please upload them as a gist or directly into this editor.
 - Detailed reproduction steps: sometimes I can spot a bug pretty easily, but often it's much more obscure. The more information I have to help reproduce it, the quicker it'll get fixed.

In survival I attempted to form a Mining turtle and would not generate result in result window.
Turtle + Unused Diamond Pick = NIL

In creative tried , same issue
In creative tried Felling Turtle (Unused diamond axe + turtle basic) - NIL
Tried with Golden turtle = NIL.

**Comment by Lemmmy** [comment]
Can you specify the information that the template asked you (i.e. the Minecraft and CC:Tweaked version)?

**Comment by SquidDev** [comment]
I suspect this is a mod compatibility issue. I've had a couple of mentions of this, but not been able to reproduce with just CC.

I suspect you'll have to narrow this down the old fashioned way - remove a bunch of mods at a time until you can find the one causing it.

**Comment by Lupus590** [comment]
If you are using a mod pack then report it to the pack authors.

**Comment by NS4W** [comment]
Will do ,

Using FTB DIREWOLF20

Thanks again


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by natiiix** [comment]
Same as @NS4W here. I'm running the latest "FTB Presents Direwolf20 1.16" modpack that's presented as the recommended modpack in the FTB app.

No matter what I try, the diamond pickaxe cannot be equipped or crafted onto the turtle. It has full durability, it is in the selected slot and there's nothing on either side of the turtle. I tried adding and removing a label from the turtle. I tried crafting a completely new turtle with a completely new diamond pickaxe with the same exact result. JEI shows all of the recipes and it puts the right items into the crafting table, but the mining turtle doesn't show up in the crafting result. Trying to equip it via a command gives "Item not equippable" and trying to equip via Lua gives "Not a valid upgrade".

**Comment by Lupus590** [comment]
An issue has been created on the mod pack's issue tracker: https://github.com/FTBTeam/FTB-Presents-Direwolf20-1.16/issues/25

**Comment by Lupus590** [comment]
Just attempted to reproduce on DW20 1.16 v1.1.2 and I could not reproduce it. I managed to craft a mining turtle with a vanilla crafting table,

**Comment by NS4W** [comment]
I tried in a vanilla crafting table , as well as in inventory.

Tried in creative mode as well as Survival . I haven’t tried in the new update as of this AM . I will now.

Thank you for your time. I will let you know If it persists .

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by NS4W** [comment]
Just tried again with :
FTB presents Direwolf20 1.16-1.12 (installed 22DEC20) FTB launcher

I attempted to craft in a vanilla crafting table using the + shortcut in the recipe. The items will place in the inventory but there is no Result (Mining turtle) generated.

I successfully created a digging turtle. But was unable to swap out to diamond pick axe.

It has been a while since I played CC is it gated  behind another block perhaps ?



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
